lib/content-services/src/lib/tag/tags-creator/tags-creator.component.ts

```diff
@@ -219,6 +219,14 @@ export class TagsCreatorComponent implements OnInit, OnDestroy {
         return this._tagNameControl;
     }
 
+    get isNameTagsVisible(): boolean {
```

Comment:
provide code documentation comments for the public api you added,

```
// Returns `true` if ....
get isNameTagsVisible(): boolean 
```

Reply:
I have updated the comments